third-party/BUILD.bazel

```diff
@@ -25,10 +25,7 @@ cc_library(
         "//conditions:default": [],
     }),
     visibility = ["//visibility:public"],
-    deps = select({
```

Comment:
Would this cause us to always link in the entire Rust standard library, even on Unix?

Reply:
(if this only affects workerd-cxx and not downstream projects, it matters a lot less of course)